Valid.java

```diff
@@ -0,0 +1,75 @@
+package Task_8;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Class consists of a method, which make a list of commands, input commands, check commands and exit from program
+ *
+ * @author Bogatko Irina
+ * @version 1.0
+ * @since 12.10.2016
+ */
+public class Valid {
+
+    /**
+     * Method, which make a list of commands
+     */
+    public ArrayList<Commands> commandsBuilder() {
+        ArrayList<Commands> allCommands = new ArrayList<>();
+        allCommands.add(new CountTypes());
+        allCommands.add(new CountAll());
+        allCommands.add(new AveragePrice());
+        allCommands.add(new AveragePriceType());
+        return allCommands;
+    }
+
+    /**
+     * Method, which input commands
+     */
+    public String inputCommand() {
+        String command;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter command: ");
+        command = sc.nextLine();
+        return command;
+    }
+
+    /**
+     * Method, which check commands
+     *
+     * @param params      entered values
+     * @param allCommands list of commands
+     */
+    public void checkCommands(ArrayList<Goods> params, ArrayList<Commands> allCommands) {

```

Comment:
This method doesn't check the command - it seems to execute commands. Unsuitable name